Pipfile

```diff
@@ -18,20 +18,16 @@ hypothesis = "==6.21.6"
 py-ecc = "==5.2.0"
 py-eth-sig-utils = "==0.4.0"
 pytz = "==2022.2.1"
-pytest = "==7.0.0"
-pytest-asyncio = "==0.18.0"
-pytest-cov = "==3.0.0"
-pytest-randomly = "==3.11.0"
```

Comment:
Having pytest here was the reason CI failed installing packages. Once removed it started working again.

Reply:
You need to update it to the versions found in `tomte`